beta/tests/requirements.txt

```diff
@@ -1 +1,8 @@
-PyYAML
\ No newline at end of file
+PyYAML
+tensorflow_addons==0.12.0

```

Comment:
It's already in main requirements.txt, I think we can do not include it here.

Reply:
Not sure how to better deal with `tensorflow_addons`. I did not include it in the main `nncf/beta/requirements.txt`, because it is used by
`beta/examples/tensorflow/common/optimizer.py`
`beta/examples/tensorflow/classification/main.py`
`beta/tests/tensorflow/sparsity/rb/test_integration.py`

In the scenario of installation `NNCF TF` + `TF examples` and usage of examples, `tensorflow_addons` can be used.
On the other side, in the case of the installation `NNCF TF` + `TF tests` and running just tests, `tensorflow_addons`  also can be used.

@as-suvorov, does `tensorflow_addons` approved for including into NNCF? If yes, we can include it in `NNCF TF` requirements to avoid duplicating.

Reply:
The current solution is ok for me. `tensorflow_addons` is not a dependency for NNCF.